corehq/ex-submodules/pillowtop/pillow/interface.py

```diff
@@ -300,14 +314,14 @@ def _record_datadog_metrics(self, changes_chunk, processing_time):
         # processing_time per change
         datadog_histogram(
             'commcare.change_feed.processing_time',
-            processing_time / len(changes_chunk),
-            tags=tags + ["chunk_size:".format(str(len(changes_chunk)))]
+            processing_time / change_count,
+            tags=tags + ["chunk_size:".format(str(change_count))]
```

Comment:
I think that this tag doesn't work because there's no `{}` in the string

also shouldn't `format` handle converting an int to a string?

Reply:
True, is this tag actually used anywhere?

Reply:
since its broken, i would assume it is not, so you could probably remove it

Reply:
59e64cc1795